src/components/common/Nav/Nav.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { NavStyled } from "./Nav.styled";
+import { NavItem } from "./NavItem";
+import { NavProps } from "./type";
+
+export const Nav: React.FC<NavProps> = ({ navItems }) => {
+  return (
+    <NavStyled>
+      <ul>
+        {navItems.map((item) => (
+          <NavItem {...item} />
+        ))}
```

Comment:
Personally, I prefer passing props one by one, but it is not that important

Reply:
No key prop also

Reply:
yeah, you're right, I'll add the key prop but I think I'll keep passing prop as a whole object (because it would be easier to add or remove props later)